invenio_rdm_records/services/permissions.py

```diff
@@ -147,8 +148,20 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     can_delete_draft = can_curate
     # Allow creating a new version of an existing published record.
     can_new_version = can_curate
+    can_publish_restricted = (
+        can_manage[1:]
```

Comment:
IMHO I would suggest to explicitly set the permissions if they cannot be fully reused.
```suggestion
        RecordCommunitiesAction("curate"),
        SystemProcess(),
```